pyproject.toml

```
@@ -64,6 +64,7 @@ docs = [
     "sphinxext-opengraph",
     "breathe"
 ]
+dev = ["mqt.ddsim[tnflow, tests, coverage, docs]"]
```

```suggestion
dev = ["mqt.ddsim[tnflow, coverage, docs]"]
```
Tests is superfluous here as it is implicitly covered by `coverage`

---

Yes, that was intentional. Maybe it's been superseeded by now, but doesn't python follow the mantra "explicit is better than implicit"?

---

In the end, it doesn't really matter and I have no particularly hard feelings on this. The specification explicitly supports recursive dependencies, so they might as well be used. Especially if the extra is defined a couple lines above in the same file.

---

Mh... okay, I'll remove it.